examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1153,10 +1153,10 @@ GLTFExporter.prototype = {
 		 */
 		function processMesh( mesh ) {
 
-			var cacheKey = mesh.geometry.uuid + ':' + mesh.material.uuid;
-			if ( cachedData.meshes.has( cacheKey ) ) {
+			var meshCacheKey = mesh.geometry.uuid + ':' + (Array.isArray(mesh.material) ? mesh.material.map(m => m.uuid).join(":") : mesh.material.uuid);
```

Comment:
We don't usually have arrow functions in these examples, do you mind writing this out with a loop?

Reply:
Done